README.md

```diff
@@ -325,6 +325,7 @@ It is possible to integrate this VPC module with [terraform-aws-transit-gateway
 | dhcp\_options\_netbios\_node\_type | Specify netbios node\_type for DHCP options set (requires enable\_dhcp\_options set to true) | `string` | `""` | no |
 | dhcp\_options\_ntp\_servers | Specify a list of NTP servers for DHCP options set (requires enable\_dhcp\_options set to true) | `list(string)` | `[]` | no |
 | dhcp\_options\_tags | Additional tags for the DHCP option set (requires enable\_dhcp\_options set to true) | `map(string)` | `{}` | no |
+| disable\_igw\_creation | Controls if an Internet Gateway is created for public subnets and the related routes that connect them. | `bool` | `false` | no |
```

Comment:
Let's change the name to the opposite which matches the rest of conditional arguments - `create_igw`.

Reply:
done, should this be added for the egress IGW as well?

Reply:
yes, probably. It is a good idea to control everything in general.

Reply:
added a different flag for egress igw and related routes to give a more granular control